### DISCUSSION TITLE
Required AAS  adaption according to Industry Core 1.0.0

### DISCUSSION THREAD

**Discussion by mkanal** [eclipse-tractusx/item-relationship-service#402] [General]
# Background 
Industry Core defined the AAS attribute 'digitalTwinType' to detect BOMLifecycle. 
This parameter is configurable within the DTR. See concept. 
DTR Team will not implement this as a default in DTR. @thomas-henn 
This implies that any data provider which is conformed with Industry Core and providing a dDTR has to configure the AAS parameter in own responsibility.

This has to be coordinated within the E2E in order to test the industry core with data provider and data consumers, 

# Procedure: 
IRS communicated that we are building on the Industry Core changes. 
We are actively communicating in the E2E and PO Sync that we expect these changes to be completed for the E2E test phase.

# Link 
- [Concept](https://github.com/catenax-ng/tx-item-relationship-service/blob/chore/%23367-industry-core-changes/docs/concept/%23367-adapt-to-changes-of-industry-core/%23367-adapt-to-changes-industry-core.md#changes-in-aas-handling)

# Reference DTR 
https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/values.yaml#L51
https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/README.md#:~:text=externalSubjectIdWildcardAllowedTypes

# Impact 

## E2E Testing Phase 
- [ ] @ds-alexander-bulgakov  9.2.2024 Alignment of Testscenarios testing changes in DTR and AAS with Testmanagement and Testmanager 

## PO Sync 
- [ ] @jzbmw & @mkanal  5.2.2024 PO Sync communication

**Comment by mkanal** [comment]
@jzbmw 
Question is if we should support both kinds of data providers, those which are conformed with Industry Core and those which are not. So we have different options to detect BOMLifecycle. What is your opinion on this?

**Reply by jzbmw** [reply]
Currently also regarding the versioning discussions going on, the answer is yes
